scheduler/src/cook/schema.clj

```diff
@@ -1001,6 +1001,20 @@ for a job. E.g. {:resources {:cpus 4 :mem 3} :constraints {\"unique_host_constra
    {:db/id (d/tempid :db.part/user)
     :db/ident :resource.type/mem
     :resource.type/mesos-name :mem}
+   {:db/id (d/tempid :db.part/user)
+    :db/ident :resource.type/disk
+    ;TODO: Check if resource.type/mesos-name is needed
```

Comment:
Is this TODO done?

Reply:
yeah, we need it for the "get-all-resource-types" function